### PR TITLE
Add config flow instructions to version integration

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Add config flow to version integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54642
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/version.markdown [hunk 1]
```diff
@@ -15,44 +15,9 @@ ha_platforms:
   - sensor
```
Comment (frenck) [inline source/_integrations/version.markdown:15]:
```suggestion
  - sensor
ha_config_flow: true
```

File: source/_integrations/version.markdown [hunk 2]
```diff
@@ -15,44 +15,9 @@ ha_platforms:
...
 ---
 
-The `version` sensor platform that can display the current Home Assistant versions.
+The `version` integration can display the current Home Assistant versions.
```
Comment (frenck) [inline source/_integrations/version.markdown]: Removing the backticks, as its no longer used in configuration :)

```suggestion
The Version integration can display the current Home Assistant versions.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/version.markdown (hunk 2 above)
Comment (ludeeus) [reply source/_integrations/version.markdown]: `{{ page.title }}` :smile:

Comment (frenck) [reply source/_integrations/version.markdown]: Sure, keeping stuff normal readable is a good thing as well :)

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**